### ISSUE TITLE
Original "pseudo :before" CSS-based fa implementation not possible?

### ISSUE THREAD

**Issue by CNovakPB** [FortAwesome/ember-fontawesome#131]
I believe I already know the answer to this question, but I want to be sure before I start down a bad path...

Because ember-fontawesome uses the "SVG JavaScript Core," and not a woff file, that means there is no way for your addon to expose the "old way" of implementing fontawesome. IE: using HTML tags and applying fontawesome classes, such as "fas fa-cog" with a pseudo :before class and content: [\unicode value], correct?

I'm asking because a member of my team has installed ember-fontawesome in our main project, but I have a usecase where I need to pass a classname string to one of our purpose-built addons to dynamically apply a glyph and/or related styles. Because I want the Ember addon I'm developing to be compartmentalized and agnostic to whatever font library we're using atm, I can't implement your <FaIcon @icon="coffee" @size="4x" /> component in the addon. BUT, because ember-fontawesome doesn't use a woff file, I can't implement fontawesome icons in the traditional way either. That means I have to manually import the original fontawesome woff file and setup my own fa classes, etc, and fontawesome will (essentially) be imported into our final app twice.

Please let me know if I have missed something, thanks!

TLDR; there is no way to implement fontawesome glyphs in the "old" css pseudo :before method with ember-fontawesome.

**Comment by robmadole** [comment]
Hi @CNovakPB. This library, like you said, is for the SVG JavaScript core. We don't support mixing one method with another (using SVG and webfonts). Sorry but this is just too difficult to support.

You might have better luck if you just switch to webfonts only and forego the use of ember-fontawesome. You will, however, lose out on the features that are provided in the SVG version.

**Comment by CNovakPB** [comment]
Yep, understood. Thanks for clarifying!
